neurow/config/runtime.exs

```diff
@@ -1,8 +1,23 @@
+# Executed at execution time
+
 import Config
 
-config :logger, :console,
-  format: "$time $metadata[$level] $message\n",
-  level: String.to_atom(System.get_env("LOG_LEVEL") || "info")
+case System.get_env("LOG_FORMAT") do
+  log_format when log_format in ["TEXT", nil] ->
+    config :logger, :console,
+      metadata: [:mfa],
+      format: "$time $metadata[$level] $message\n",
+      level: String.to_atom(System.get_env("LOG_LEVEL") || "info")
+
+  "ECS" ->
+    config :logger, :console,
+      metadata: :all,
+      level: String.to_atom(System.get_env("LOG_LEVEL") || "info"),
+      format: {Neurow.EcsLogFormatter, :format}
+
+  other_format ->
```

Comment:
**Suggestion** 🤔 : should we rathor log and use default ?

Reply:
The idea is to make it crash for any configuration issue.